.github/workflows/ci-build-test.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: ci-build-test
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2.4.0
+
+      - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v2.2.0
+        with:
+          go-version: '1.19'
+          check-latest: true
+          cache: true
+
+      - name: build
+        run: make binary
+
+  test:
+    name: test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2.4.0
+
+      - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v2.2.0
+        with:
+          go-version: '1.19'
+          check-latest: true
+          cache: true
+
+      - name: build
+        run: make test
```

Comment:
Do the tests actually pass currently?
In my experience https://github.com/chainguard-dev/vex/blob/main/pkg/attestation/attestation_test.go#L29 is broken

Reply:
OK, I checked it and some of the tests are not up to date with the last minute changes, I'm fixing them to enable the ci jobs

Reply:
it is broken now but we are working to fix it